polyfill/lib/absolute.mjs

```diff
@@ -0,0 +1,138 @@
+import { ES } from "./ecmascript.mjs";
+import { SLOT_EPOCHNANOSECONDS } from "./slots.mjs";
+
+export function Absolute(epochNanoSeconds) {
+  if (!(this instanceof Absolute)) return new Absolute(epochNanoseconds);
+  this[SLOT_EPOCHNANOSECONDS] = epochNanoSeconds;
+}
+
+Absolute.prototype.getEpochSeconds = function getEpochSeconds() {
+  let epochNanoSeconds = this[SLOT_EPOCHNANOSECONDS];
+  let epochSecondsBigInt = epochNanoSeconds / BigInt(1_000_000_000);
```

Comment:
why not use bigint syntax?

Reply:
Because the transformation I use can understand BigInt syntax (123n) and the Number Separator proposal (1_000) but not the combination thereof. So this is purely for convenience.

Reply:
I'd really discourage this pattern as well, though it's a minor point. Did you file an issue against whoever upstream fails to support the combination of BigInt and numeric separators?

Reply:
Changed and fixed tooling